torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -15,6 +15,64 @@ constexpr auto kNoTimeout = std::chrono::milliseconds(0);
 
 namespace c10d {
 
+enum OpType {
+  BROADCAST = 0,
+  ALLREDUCE = 1,
+  ALLREDUCE_COALESCED = 2,
+  REDUCE = 3,
+  ALLGATHER = 4,
+  ALLGATHER_BASE = 5,
+  ALLGATHER_COALESCED = 6,
+  GATHER = 7,
+  SCATTER = 8,
+  REDUCE_SCATTER = 9,
+  ALLTOALL_BASE = 10,
+  ALLTOALL = 11,
+  SEND = 12,
+  RECV = 13,
+  RECVANYSOURCE = 14,
+  BARRIER = 15,
+};
+
+static std::string opTypeToString(OpType opType) {
```

Comment:
nit: Any reason to declare it static instead of putting it into an anonymous namespace?